controllers/subscription.ts

```diff
@@ -0,0 +1,81 @@
+import { CustomRequest, CustomResponse } from "../types/global";
+const { addOrUpdate } = require("../models/users");
+const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+const nodemailer = require("nodemailer");
+const config = require("config");
+const emailSubscriptionCredentials = config.get("emailSubscriptionCredentials");
+
+export const subscribe = async (req: CustomRequest, res: CustomResponse) => {
+  try {
+    const { email } = req.body;
```

Comment:
Is this safe from a security point of view?

Reply:
Yes, it is same as we are validating this data at route level with the help of a middleware.